consensus/src/main/java/org/apache/iotdb/consensus/ratis/RatisConsensus.java

```diff
@@ -575,6 +583,50 @@ public ConsensusGenericResponse triggerSnapshot(ConsensusGroupId groupId) {
     return ConsensusGenericResponse.newBuilder().setSuccess(reply.isSuccess()).build();
   }
 
+  private void triggerSnapshotByCustomize(ConsensusConfig config) {
```

Comment:
maybe we should add synchronized to method signature. Consider this: this method is not finished when 60s past and another call started.